src/plots.py

```diff
@@ -325,3 +325,31 @@ def plot_donut(df, col_color):
             },
         },
     )
+
+def plot_line(df,col_x, col_y,col_color):
+    st.vega_lite_chart(
+        data= df,
+        spec={
```

Comment:
@utopian-monkey Could you add `**CONFIG_MAIN,` such that the plot size is consistent with the rest of the tabs?

Reply:
We could also add interactivity like on the other plots using `"tooltip": True` and `"point": True` (maybe try to reduce point size if too many)